tiny_dnn/core/kernels/conv2d_op_avx.h

```diff
@@ -375,6 +390,7 @@ void avx_conv2d_5x5_kernel(const core::conv_params &params,
         size_t cnt  = (out_area - headSize) / 8;
         double *pa2 = pa + headSize;
         for (size_t i = 0; i < cnt; ++i) {
+          // TODO: segfault here even though adresses are ok
           _mm256_store_pd(&pa2[i * 8 + 0], b2);
```

Comment:
I still haven't figured out why segfault happens here. Pointer seems to be legal.